### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by cloudprogramm**

### CODE REVIEW

**Review by Oleksandr563487953 — APPROVED** [review]

File: tests/test_main.py [hunk 1]
```diff
@@ -1 +1,23 @@
-# write your code here
+from unittest import mock
+from app.main import cryptocurrency_action
+
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_if_predicated_exchange_rate_bigger_then_5(mocked_exchange_rate):
+    mocked_exchange_rate.return_value = 1.49
+
+    assert cryptocurrency_action(1.49) == "Sell all your cryptocurrency"
```
Comment (MasterpieceElbow) [inline tests/test_main.py:9]: This tests should not pass. Update task please.

File: tests/test_main.py [hunk 2]
```diff
@@ -1 +1,23 @@
...
+    mocked_exchange_rate.return_value = 2
+
+    assert cryptocurrency_action(1.5) == "Buy more cryptocurrency"
+
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_if_predicated_exchange_rate_difference_low(mocked_exchange_rate):
+    mocked_exchange_rate.return_value = 2.1
+
+    assert cryptocurrency_action(2) == "Do nothing"
```
Comment (MasterpieceElbow) [inline tests/test_main.py:23]: Add boundary conditions, you don't check when `get_exchange_rate_prediction` return value equals exactly 0.95 * value passed in `cryptocurrency_action`.

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]

File: tests/test_main.py [hunk 3]
```diff
@@ -1 +1,23 @@
...
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_if_predicated_exchange_rate_lower_than_5(mocked_exchange_rate):
+    mocked_exchange_rate.return_value = 2
+
+    assert cryptocurrency_action(1.5) == "Buy more cryptocurrency"
+
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_if_predicated_exchange_rate_difference_low(mocked_exchange_rate):
```
Comment (IvanRamyk) [inline tests/test_main.py:20]: Could you also add a test case for 0.95?

**Review by IvanRamyk — CHANGES_REQUESTED** [review]
